src/transformers/integrations.py

```diff
@@ -1313,6 +1313,8 @@ class ClearMLCallback(TrainerCallback):
         ClearML project name.
     - **CLEARML_TASK** (`str`, *optional*, defaults to `Trainer`):
         ClearML task name.
+            CLEARML_LOG_MODEL (`bool`, *optional*, defaults to `False`):
+                Whether to log models as artifacts during training.
```

Comment:
As you can see above we just merged a PR that slightly changed the doc of this class. Could you adapt your addition?
```suggestion
    - **CLEARML_LOG_MODEL** (`bool`, *optional*, defaults to `False`):
        Whether to log models as artifacts during training.
```